.eslintrc

```diff
@@ -2,7 +2,34 @@
     "extends": "airbnb",
     "rules": {
         // Override any settings from the airbnb configuration
-        "indent": [2, 4],
-        "comma-dangle": 0
+        // "indent": [2, 4],
+        "comma-dangle": 0,
+        // To fix
+        "brace-style": 0,
+        "camelcase": 0,
```

Comment:
`0` is the same as `off`, it means "ignore this rule". So we're turning off rules that were turned on in the airbnb style guide.